osu.Game.Rulesets.Karaoke/Edit/Lyrics/Rows/Edit/Carets/DrawableLyricInputCaret.cs

```diff
@@ -60,6 +62,12 @@ public DrawableLyricInputCaret(bool preview)
                     ReleaseFocusOnCommit = false,
                     NewCommitText = text =>
                     {
+                        if (lyricTextChangeHandler.IsSelectionsLocked())
+                        {
+                            editLyricRowState.TriggerDisallowEditEffect();
+                            return;
+                        }
+
```

Comment:
Maybe need the discussion in the https://github.com/karaoke-dev/karaoke/issues/1563#issuecomment-1246788132 but  seems OK to let it merge now.